moveit_ros/hybrid_planning/test/config/hybrid_planning_manager.yaml

```diff
@@ -0,0 +1 @@
+planner_logic_plugin_name: "moveit_hybrid_planning/ReplanInvalidatedTrajectory"
```

Comment:
Is it possible to combine all these config files into 1 file? Do we even want to?

Reply:
I guess this is debatable. My thought was that it might be more clearly arranged if I use one config file per component node especially when plugins are used, that have many parameters. But I don't have a strong opinion on this and I also see advantages in using one config file. It would be great to hear the opinion of other reviewers on this